src/common_values.jl

```diff
@@ -248,7 +326,7 @@ The divergence of a vector valued functions in the quadrature point ``\\mathbf{x
 
 where ``\\mathbf{u}_i`` are the nodal values of the function.
 """
-@inline function function_divergence{dim, T}(fe_v::AbstractFEValues{dim}, q_point::Int, u::Vector{Vec{dim, T}})
+function function_divergence{dim, T}(fe_v::ScalarValues{dim}, q_point::Int, u::Vector{Vec{dim, T}})
```

Comment:
Yes allow for the previous behavior. I don't want all my code to break and it is technically better performance even if you have to do some algebra to get the corresponding expression from the weak form. :P